tests/integration/llm/client.py

```diff
@@ -1356,6 +1375,86 @@ def test_handler(model, model_spec):
                 awscurl_run(req, spec.get("tokenizer"), batch_size)
 
 
+def log_awscurl_benchmark(metric_name: str,
+                          benchmark_name: str = "benchmark.json") -> None:
+    with open(benchmark_name, "r") as f:
+        raw_metrics = json.load(f)
+        metrics = list()
+        metrics.append({
+            "MetricName": f"{metric_name}_p50Latency",
+            "Unit": "Milliseconds",
+            "Value": raw_metrics["p50Latency"]
+        })
+        metrics.append({
+            "MetricName": f"{metric_name}_p90Latency",
+            "Unit": "Milliseconds",
+            "Value": raw_metrics["p90Latency"]
+        })
+        metrics.append({
+            "MetricName": f"{metric_name}_p50TimeToFirstByte",
+            "Unit": "Milliseconds",
+            "Value": raw_metrics["p50TimeToFirstByte"]
+        })
+        metrics.append({
+            "MetricName": f"{metric_name}_p90TimeToFirstByte",
+            "Unit": "Milliseconds",
+            "Value": raw_metrics["p90TimeToFirstByte"]
+        })
+        metrics.append({
+            "MetricName": f"{metric_name}_tokenThroughput",
+            "Unit": "Count/Second",
+            "Value": raw_metrics["tokenThroughput"]
+        })
+        metrics.append({
+            "MetricName": f"{metric_name}_tps",
+            "Unit": "Count/Second",
+            "Value": raw_metrics["tps"]
+        })
+        metrics.append({
+            "MetricName": f"{metric_name}_tokenPerRequest",
+            "Unit": "Count",
+            "Value": raw_metrics["tokenPerRequest"]
+        })
+        LOGGER.info(f"{metric_name}")
+        LOGGER.info(f"raw metrics: {raw_metrics}")
+        command = f'aws cloudwatch put-metric-data --namespace "serving_handler" ' \
+                  f'--region "us-east-1" --metric-data \'{json.dumps(metrics)}\''
+        LOGGER.info(command)
+        sp.call(command, shell=True)
+
+
+def run_rb_handler_performance(benchmark_name, model_spec, req):
+    for batch_size in model_spec["batch_size"]:
+        high_concurrency = batch_size > 16
+        metric_name = f"{benchmark_name}-high-concurrency" if high_concurrency else benchmark_name
```

Comment:
I'd prefer we keep the metric name consistent for all batch_sizes/concurrency, and instead use a dimension for batch size. I'm worried that if we change the batch_size test cases above, we'll lose insight into the behavior differences across more granular batch size tests

Reply:
I will make an update to just have concurrency in the name, this along with the note above make sense. I think my longer range goal is to be able to also handle multi-modal handler performance like this, so there might be some discussion around metric name there as well.